src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -88,21 +97,37 @@ export const ProgressStep = ({ ...props }) => {
 
   return (
     <li className={classes}>
-      {currentSvg || completeSvg || incompleteSvg}
-      <ProgressStepLabel className={`${prefix}--progress-label`}>
-        {label}
-      </ProgressStepLabel>
-      {componentsX &&
-      secondaryLabel !== null &&
-      secondaryLabel !== undefined ? (
-        <p className={`${prefix}--progress-optional`}>{secondaryLabel}</p>
-      ) : null}
-      <span className={`${prefix}--progress-line`} />
+      <div
```

Comment:
it appears that the experimental version of this component is broken with this change. I think we may have to incorporate the new markup and add new style rules to the vanilla component before we can merge this into the React repo